src/permissions.md

```diff
@@ -0,0 +1,69 @@
+# Permissions
+
+Assets are protected using a permission system.
+
+Permissions are either specified  by the user at asset registration time, or they are automatically inherited from dependent assets (see “Permission inheritance”).
+
+## Public vs Authorized_Ids
+
+Permissions are structured into two properties:
+
+- *Public* (bool): if True, the asset is downloadable/processable *by all nodes*  in the channel.
+- *Authorized ids* (list of strings): a *list of nodes* allowed to process/download the asset.
+
+Using this structure, there are 3 possible use-cases:
+
+- the asset is downloadable or processable by **all nodes** in the network => `Public = True`
+- the asset is downloadable or processable by **a set of nodes** in the network, listed by their identity -> `Public = True, Authorized_Ids = [NodeA, NodeB]`
```

Comment:
The example does not match the description:
```suggestion
- the asset is downloadable or processable by **a set of nodes** in the network, listed by their identity -> `Public = False, Authorized_Ids = [NodeA, NodeB]`
```

Reply:
Good catch!